### PR TITLE
add task solution

### PR DESCRIPTION
**Pull request by KristinaMelnyk**
- [DEMO LINK](https://KristinaMelnyk.github.io/layout_moyo-header/)
- [TEST REPORT LINK](https://KristinaMelnyk.github.io/layout_moyo-header/report/html_report/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -1,14 +1,37 @@
...
+      <nav class="menu">
+        <ul class="menu_bar">
+          <li class="list"><a href="#" class="link blue is-active">Apple</a></li>
+          <li class="list"><a href="#" class="link">Samsung</a></li>
+          <li class="list"><a href="#" class="link">Smartphones</a></li>
+          <li
+          class="list">
+            <a href="#" class="link blue"
+          data-qa="hover">Laptops & Computers</a>
+          </li>
```
Comment (etojeDenys) [inline src/index.html:28]: Pasted Graphic

**Review by etojeDenys — CHANGES_REQUESTED** [review]
try to fix the tests or don't hesitate to ask for some help in the chat

**Comment by KristinaMelnyk** [comment]
I fixed all tests

**Review by volodymyr-soltys97 — CHANGES_REQUESTED** [review]
Good job 👍 
Let's improve your code

File: src/index.html [hunk 2]
```diff
@@ -1,14 +1,57 @@
...
+
+          <li class="nav_item">
+            <a
+            href="#"
+            class="nav__link"
+            data-qa="hover"
+            >
+            laptops & computers
+            </a>
```
Comment (volodymyr-soltys97) [inline src/index.html:35]:
```suggestion
            <a
              href="#"
              class="nav__link"
              data-qa="hover"
            >
              laptops & computers
            </a>
```

**Review by volodymyr-soltys97 — APPROVED** [review]
Good job 👍

**Pull request closed without merging**